.github/ISSUE_TEMPLATE/request-a-feature.md

```diff
@@ -27,20 +31,23 @@ If you have a feature suggestion for EssentialsX, read the following tips:
 3.  Check whether it has already been asked or added.
       You can search the issue tracker to see if your feature has already been
       requested at https://github.com/EssentialsX/Essentials/issues. You can
-      also check the changelogs at https://github.com/EssentialsX/Essentials/releases
-      to see if the feature was recently added.
+      also check the changelogs to see if the feature was recently added:
+      https://github.com/EssentialsX/Essentials/releases
```

Comment:
Maybe provide links to development builds instead/alongside of releases?
```suggestion
      also check the changelogs to see if the feature was recently added:
      https://github.com/EssentialsX/Essentials/releases
      If you still can't find the feature, try using development builds:
      https://essentialsx.net/downloads.html
```

Reply:
Development builds don't provide properly-documented explanations of changes, whereas changelogs always have full documentation of new features.